packages/firebase_storage/android/src/main/java/io/flutter/plugins/firebase/storage/FirebaseStoragePlugin.java

```diff
@@ -45,12 +45,35 @@ public void onMethodCall(MethodCall call, final Result result) {
       case "StorageReference#getData":
         getData(call, result);
         break;
+      case "StorageReference#deleteFile":
+        deleteFile(call, result);
+        break;
       default:
         result.notImplemented();
         break;
     }
   }
 
+  private void deleteFile(MethodCall call, final Result result) {
+    String path = ((Map<String, String>) call.arguments).get("path");
```

Comment:
```dart
String path = call.argument("path");
```

Reply:
Ah, sorry about that one. For some reason intellij doesn't give me completion when I open this project in it. (Maybe it's a bug in plugin?). Fixed now.